bootstrap/cmd/kfctl/cmd/delete.go

```diff
@@ -68,4 +72,12 @@ func init() {
 		log.Errorf("couldn't set flag --%v: %v", string(kftypes.VERBOSE), bindErr)
 		return
 	}
+
+	deleteCmd.Flags().Bool(string(kftypes.DELETE_STORAGE), false,
```

Comment:
why is it false by default?

Reply:
from Jeremy storage deletion is optional.